cpp/src/transform/bools_to_mask.cu

```diff
@@ -42,11 +42,11 @@ bools_to_mask(column_view const& input,
     auto input_device_view_ptr = column_device_view::create(input, stream);
     auto input_device_view = *input_device_view_ptr;
     auto pred = [] __device__ (bool element) {
-        return element;
+        return !element;
     };
     if(input.nullable()) {
-        // Nulls are considered false
-        auto input_begin = make_null_replacement_iterator<bool>(input_device_view, false);
+        // Nulls are considered true
+        auto input_begin = make_null_replacement_iterator<bool>(input_device_view, true);
```

Comment:
This definitely doesn't look right to me. At the very least, `mask_to_bools` has inverted logic.

Reply:
This is correct, nulls are 0, valids are 1 in the case of our nullmask